.env.template

```diff
@@ -3,7 +3,5 @@ FLASK_APP=todo_app/app
 FLASK_ENV=development
 
 # Change the following values for local development.
-TRELLO_API_KEY=
-TRELLO_API_TOKEN=
-TRELLO_URL=
-TRELLO_BOARD_ID=
+MONGO_CON_STRING=
+MONGO_DB_NAME=
```

Comment:
Well remembered to update all of these bits!

Reply:
Thanks Jack